app/src/components/ModeratorOnly.tsx

```diff
@@ -2,9 +2,9 @@ import * as React from 'react';
 import Container from 'react-bootstrap/Container';
 
 interface GovernanceProps {
-  drizzle: any;
-  drizzleState: any;
   children: React.ReactNode | React.ReactNode[];
+  contracts: any;
+  accounts: Array<string>;
```

Comment:
Can these either reference interface so we can fix it all in one, or be replaced by a more accurate typing?

Reply:
cant use the contract interface we had previously because it used a 'methods' list that is no longer present. the methods are now directly accessed, meaning we'd need one interface per each of the four contracts, since every function would have to be in the interface. 

the type from @truffle/contract is really hard to get at.

Reply:
Why not use web3.eth.contract?

Reply:
i thought i was using web3 contracts but it turns out @truffle/contract produces its own very similar wrapper around web3, where the types inaccessible as far as i can tell. is this gonna cause issues with universal login / formatic do you think? if so i can rewrite to use web3.eth.contract

Reply:
i figured out how to access the trufflecontract type at last